cmd/session-migrate.go

```diff
@@ -37,6 +37,9 @@ func migrateSessionV7ToV8() {
 			fatalIf(err.Trace(sid), "Unable to load version `7`. Migration failed please report this issue at https://github.com/minio/mc/issues.")
 		}
 
+		// Close underlying session data file.
+		sV7.DataFP.Close()
```

Comment:
if we are closing the session data file right after returning from `loadSessionV7`, why can't we close it at the end of `loadSessionV7`?